Directory.Packages.props

```diff
@@ -24,10 +24,10 @@
     <PackageVersion Include="Microsoft.VisualStudioEng.MicroBuild.Core" Version="1.0.0" />
     <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
     <PackageVersion Include="Microsoft.VisualStudio.Internal.MicroBuild.NonShipping" Version="$(MicroBuildVersion)" />
-    <PackageVersion Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.5.21" />
+    <PackageVersion Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.6.29-alpha" />
     <PackageVersion Include="Microsoft.Windows.CsWin32" Version="0.2.188-beta" />
     <PackageVersion Include="NuGet.Protocol" Version="6.4.0" />
-    <PackageVersion Include="StreamJsonRpc" Version="2.15.3-alpha" />
+    <PackageVersion Include="StreamJsonRpc" Version="2.15.12-alpha-g3116407ec1" />
```

Comment:
I will update this when https://github.com/microsoft/vs-streamjsonrpc/pull/886 merges and we have an official build to consume.